SevenSwitch.swift

```diff
@@ -201,12 +221,18 @@ import QuartzCore
     private var isAnimating: Bool = false
     private var userDidSpecifyOnThumbTintColor: Bool = false
     private var switchValue: Bool = false
+    lazy var knobSize: CGSize = {
+        let height = self.frame.height - 2
+        let width = self.frame.height - 2
+        let size = CGSize (width: width, height: height)
+        return size
+    } ()
     
     /*
     *   Initialization
     */
     public convenience init() {
-        self.init(frame: CGRectMake(0, 0, 50, 30))
+        self.init(frame: CGRectMake(0, 0, 50, 31))
```

Comment:
Because it is a mistake in the library. Also, I have to change 50 to 51.
Apple has 51x31 as standard slider dimensions, I've checked in View Debugger.
![screen shot 2015-06-18 at 12 36 49](https://cloud.githubusercontent.com/assets/8013017/8229437/fa98e006-15b6-11e5-96ac-592c95a31241.png)